content/en/xtc/privacy/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Privacy"
+
+type: docs
+
+weight: 900
+
+description: How XTC processes your personal data.
+---
+
+You can find all the details about the processing of personal data on Xceptance websites in the [Xceptance Privacy Statement](https://www.xceptance.com/en/contact/privacy.html). This is the condensed version of how XTC is handling personal data:
+
+### Cookie Usage
+We use only necessary cookies to make our site work. Necessary cookies enable core functionality such as security, network management, and accessibility. You cannot disable these without breaking the functionality.
+
+### Data Storage (Hosting)
+All resources are hosted at _xtc.xceptance.com_. 
+
+Data is stored on hardware provided by Google Cloud, but not shared with Google. 
+
+### Data Privacy (Analytics/Tracking)
+XTC includes no external analytics services or tracking.
+
+None of your data generated by or entered in XTC will be shared with third parties.
+
+In case you chose to use an [external login provider]({{< relref "../basics/040-user-accounts/#external-login-providers-sso" >}}) for authentication you will exchange data with your login provider (Google or Microsoft) on external servers.
+
+### Data Processing (Account Data)
+In order to use XTC, you need to create an account. The information and personal data you share with us will be utilized solely for the use of XTC services. This information is: 
+* **Required:**
+    * First Name
+    * Last Name
+    * Email
+* **Optional:**
+    * Phone Number (for monitoring notifications via text message)
+
+XTC does _not_ store any of your contractual information or payment data.
+
+### Data Deletion
+There is no automated data retention for load test data in XTC. As long as an organization or project exists, this data is retained. You can delete this data yourself if needed.
+
+XTC does not do automated account management (i.e. deactivation/deletion of inactive accounts). If you need to delete your account, you can write an email to support@xceptance.com to ask for a system-wide deletion. Please note that audit logs will retain name and email address, even when the account has been deleted, for 180 days. After that, it is anonymized, e.g. John Doe -> J\*\*\* D\*\*\* and john.doe@anydomain.net -> john@\*\*\*.net. This data is kept for up to 5 years.
```

Comment:
XTC does not perform automated account management

Please note, that audit logs will retain the name and email address for 180 days, even when your account has been deleted.